hardware/raspberrypi/revision-codes/README.md

```diff
@@ -104,6 +104,7 @@ New-style revision codes in use:
 | a02082 | 3B                | 1.2      | 1GB   | Sony UK      |
 | a020a0 | CM3               | 1.0      | 1GB   | Sony UK      |
 | a020d3 | 3B+               | 1.3      | 1GB   | Sony UK      |
+| a02042 | 2B (with BCM2837) | 2.0      | 1GB   | Sony UK      |
```

Comment:
I think you meant to put `1.2` in the Revision column?

Reply:
Oh, I misunderstood the revision code, yes, 1.2 is what I meant, let me redo the pull request with that fix.